src/app/custom/enrichers/enrichers-catalog.json

```diff
@@ -191,92 +208,109 @@
+        "objectifTDM": "https://services.istex.fr/2021/12/16/detection-de-la-llangue-dun-texte/",
         "type": "other"
     },
-    {
-        "id": "post-v1-first-name-gender",
-        "url": "https://authors-tools.services.istex.fr/v1/first-name/gender",
-        "doc": "https://gitbucket.inist.fr/tdm/web-services/tree/master/authors-tools#v1%2ffirst-name%2fgender",
-        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=authors-tools%20-%20Les%20web%20services%20qui%20se%20lancent%20sur%20le%20champs%20%27authors%27#/default/post_v1_first_name_gender",
-        "objectifTDM": "https://services.istex.fr/2023/04/26/detection-de-genre/",
-        "type": "classification"
+                      {
+        "id": "post-v1-textExtract",
+        "url": "https://pdf-text.services.istex.fr/v1/pdf2txt",
+        "doc": "https://github.com/Inist-CNRS/web-services/tree/main/services/pdf-text",
+        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=pdf-text%20-%20Transforme%20un%20PDF%20en%20texte%20%C3%A0%20partir%20de%20l%27URL#/pdf-text/post-v1-pdf2txt",
+        "objectifTDM": "https://services.istex.fr/extraction-du-texte-epure-de-pdf/",
+        "type": "prétraitement"
     },
-    {
+        {
         "id": "post-v1-lemma",
         "url": "https://nlp-tools2.services.istex.fr/v1/lemma",
         "doc": "https://gitbucket.inist.fr/tdm/web-services/tree/master/nlp-tools2#v1%2flemma",
         "swagger": "https://openapi.services.istex.fr/?urls.primaryName=nlp-tools2%20-%20Outils%20de%20NLP#/NLP/post-v1-lemma",
         "objectifTDM": "https://services.istex.fr/2023/04/19/lemmatiseur_eng/",
-        "type": "other"
-    },
-    {
-        "id": "post-v1-corporate-private-public",
-        "url": "https://authors-tools.services.istex.fr/v1/corporate/private-public",
-        "doc": "https://gitbucket.inist.fr/tdm/web-services/tree/master/authors-tools/v1/corporate",
-        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=authors-tools%20-%20Les%20web%20services%20qui%20se%20lancent%20sur%20le%20champs%20%27authors%27#/default/post_v1_corporate_private_public",
-        "objectifTDM": "https://services.istex.fr/2023/09/06/detection-daffiliations-privees/",
-        "type": "affiliation"
-    },
-    {
-        "id": "post-v1-teeft-with-numbers-en",
-        "url": "https://terms-extraction.services.istex.fr/v1/teeft/with-numbers/en",
-        "doc": "https://gitbucket.inist.fr/tdm/web-services/tree/master/terms-extraction#v1%2fteeft%2fwith-numbers%2fen",
-        "swagger": "https://openapi.services.istex.fr/?urls.primaryName=terms-extraction%20-%20Extraction%20de%20termes#/terms-extraction/post-v1-teeft-with-numbers-en",
-        "objectifTDM": "https://services.istex.fr/2023/06/30/extraction-de-termes-via-teeft-nombres-compris/",
-        "type": "indexation"
-    }
+        "type": "TAL"
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** I think `type` is a key, it has to be exactly the same as in the `translations.tsv` file.
‏
```suggestion
        "type": "tal"
```